jonathan_armenta.md

```diff
@@ -0,0 +1,3 @@
+Building components that interact with users
+Need to be proificient in HTML, CSS and Javascript
+Usually developed user friendly 
```

Comment:
Creating applications and websites that are easy to use with good design are very important. Great response.